Mathlib/Order/Monotone/Basic.lean

```diff
@@ -238,42 +238,46 @@ theorem strictAntiOn_toDual_comp_iff : StrictAntiOn (toDual ∘ f : α → βᵒ
   Iff.rfl
 #align strict_anti_on_to_dual_comp_iff strictAntiOn_toDual_comp_iff
 
-protected theorem Monotone.dual (hf : Monotone f) : Monotone (toDual ∘ f ∘ ofDual : αᵒᵈ → βᵒᵈ) :=
-  swap hf
+theorem monotone_dual_iff : Monotone f ↔ Monotone (toDual ∘ f ∘ ofDual : αᵒᵈ → βᵒᵈ) := by
```

Comment:
I think it would be better to swap the two sides of the statement, to match [monotone_toDual_comp_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Monotone/Basic.html#monotone_toDual_comp_iff) (as usual, most complicated side on the LHS)

Reply:
Yes. Done.